config/style.yml

```diff
@@ -147,41 +141,17 @@ Style/GuardClause:
   Description: 'Check for conditionals that can be replaced with guard clauses'
   Enabled: false
 
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
-Style/MethodDefParentheses:
-  Description: >-
-    Checks if the method definitions have or don't have
-    parentheses.
-  Enabled: false
```

Comment:
Oh there the debates we could end with this rule, if it wasn't for pesky RSpec.

Reply:
That sounds interesting - can you explain how this relates to RSpec?

Reply:
I should have read more about it, I mistook it for: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MethodCallWithArgsParentheses